setup.py

```diff
@@ -20,6 +20,6 @@
       author_email='opesci@imperial.ac.uk',
       license='MIT',
       packages=find_packages(exclude=['docs', 'examples', 'tests']),
-      install_requires=['numpy', 'sympy', 'mpmath', 'cgen', 'codepy',
+      install_requires=['numpy', 'sympy==1.0', 'mpmath', 'cgen', 'codepy',
```

Comment:
Shouldn't this list match that in requirements.txt? Or can we just drop this @mlange05